01-introduction.Rmd

```diff
@@ -89,44 +85,61 @@ It can be seen as a part of Geography\index{geography}, which has a 2000+ year h
+Early geographers used a variety of tools including barometers, compasses and [sextants](https://en.wikipedia.org/wiki/Sextant) to advance knowledge about the world [@wulf_invention_2015]. 
+It was only with the invention of the marine [chronometer](https://en.wikipedia.org/wiki/Marine_chronometer) in 1761 that it became possible to calculate longitude at sea, enabling ships to take more direct routes, for example.
+Using a sextant was a difficult-to-learn but essential tool for navigation in the 18th century.
+Likewise, learning to program and write and share reproducible code can be hard, but is a key skill for modern geocomputation.
+<!-- Maps were predominantly hand-drawn until modern computing enabled digitisation, a process which only become widespread in the 1990s and which is still ongoing [@gold_outsidein_1996; @auffret_histmapr_2017]. -->
+
+Nowadays, researchers and practitioners have no such limitations and in some cases face the opposite problem: too much data; too many tools.
+Most phones now have a global positioning (GPS\index{GPS}) receiver.
+Sensors ranging from satellites and semi-autonomous vehicles to citizen scientists incessantly measure every part of the world.
+The rate of data produced can be overwhelming, with emerging technologies such as autonomous vehicles generating hundreds or even thousands of GB of data daily.
+Remote sensing\index{remote sensing} datasets from satellites are too large to analyze with a single computer, as outlined in Chapter \@ref(gis).
+This 'geodata revolution' drives demand for high performance computer hardware and efficient, scalable software to handle and extract signal from the noise.
+Evolving open source tools can import and process subsets from the vast geographic data stores directly, and via APIs and via interfaces to databases. \index{spatial database}
+
+With the rapidly changing hardware, software and data landscapes, it's important to choose tools that are future-proof, able to adapt.
+A major advantage of open source software is its **rate of development and longevity**, with thousands of potential contributors.
+Hundreds of people submit bug reports, suggested new features, and documentation improvements to open source projects every day, a rate of evolution that most proprietary solutions simply cannot keep up with.
+
+Another major advantage of open source software is its **interoperability**.
+While proprietary products tend to be monolithic 'empires' that are difficult to maintain (linked to the previously mentioned advantage), open source software is more like a 'federation' of modular tools that can be combined in different ways.
+This has allowed open source data science languages such as R to rapidly incorporate new developments such as interfaces to high performance visualisation libraries and file formats, while proprietary solutions struggle to keep up.
+
+Another major advantage is **reproducibility**.
```

Comment:
This paragraph starts exactly the same as the previous one...

Reply:
Woops. Thanks Jakub, will fix.

Reply:
Updated sentences fix it:

A linked advantage is **interoperability**.
While proprietary products tend to be monolithic 'empires' that are difficult to maintain (linked to the previously mentioned advantage), open source software is more like a 'federation' of modular tools that can be combined in different ways.
This has allowed open source data science languages such as R to rapidly incorporate new developments such as interfaces to high performance visualisation libraries and file formats, while proprietary solutions struggle to keep up.

Another major advantage is **reproducibility**.
Being able to replicate findings is vital for scientific research, and open source software removes an important potential barrier to others being able to check your findings or apply your methods in new places.
The combination of using tools that can be accessed by anyone for free with the ability to share code and data means that the results of your work can be checked and built upon by others, a huge advantage if you want your work to be used and cited.